test/02LessonTwoTest.js

```diff
@@ -11,6 +25,12 @@ function whichSpecies(character) {
 }
 
 function isEven(number) {
+    if(number %2 === 0){
+      return true;
+    }
+    else{
+      return false;
+    }
     // should return true is number is even (divisible by 2)
 
 }
```

Comment:
This is good. One thing to note, though, `(number % 2 === 0)` will either be **true** or **false**.

So, you can do this `return (number % 2 === 0);` Does that make sense?